packages/app/src/TeamPage.tsx

```diff
@@ -355,6 +377,23 @@ export default function TeamPage() {
                     )}
                 </>
               )}
+              <div className="my-5">
+                <h2>Sentry Integration</h2>
+                <div className="text-muted">
+                  To setup Sentry integration, copy the following Sentry DSN
```

Comment:
I think it would be more clear to paste the doc link or something to tell people what DSN is or how to update it in sentry SDK